### ISSUE TITLE
Option to use other fonts

### ISSUE THREAD

**Issue by gkomarnicki** [moezbhatti/qksms#1013]
I feel Roboto is a clearer and more readable font for QKSMS than Lato, even though the former is slightly bigger. There could be an option in settings for switching between the two fonts.

Would it also be possible to use Product Sans in the app?

**Comment by moezbhatti** [comment]
As far as I know, you're not allowed to distribute Product Sans in your own app.

And as for using Roboto, I do plan on adding an option for this. It won't be to explicitly use Roboto, rather to just use the system font.

**Comment by gkomarnicki** [comment]
Yeah I was thinking that there'd be some sort of restriction with Product Sans, though Action Launcher includes it.

What about on devices that don't use Roboto as the default font - use the system font anyway?

**Comment by dylmye** [comment]
Yes, please revert the default back to the system font or provide an option to use the system font. I've switched back to Signal for now.

**Comment by moezbhatti** [comment]
@gkomarnicki yeah, it'll use the system default. At this point there's not many devices that don't have Roboto, so it would just be unnecessarily making the app larger
